### ISSUE TITLE
The essence of poor performance of novel classes

### ISSUE THREAD

**Issue by userzhi** [xavibou/ovdsat#7]
hello, @xavibou ,i have read  this paper,  a question  i  think about  recently  is what is the essence of poor performance in novel classes, 
 two answers i guess as following:
          1）it is unable to generate region proposals for foreground in novel classes
          2)  it is able to generate region proposals for foreground in novel classes, but fail to classify correctly
I don't know whether the above two points are correct or not

**Comment by xavibou** [comment]
Hi @userzhi, you point out a very adequate question. The paper discusses this briefly in the experiments/conclusion section. There seems to be a bottleneck caused by the RPN when the target objects look very different from the ones in the training dataset. You can clearly see this in the paper results. When training the RPN with DOTA, the results on SIMD are quite good, as the types of objects are subclasses of the DOTA categories (types of planes, vehicles, etc.). Contrary to this, DIOR contains several classes that have nothing to do with the ones in DOTA, e.g. toll station, wind turbine, dam, etc. These look very different and are considered as background by the FPN, so no quality bounding boxes are generated for them, causing a lot of missed detections.

In conclusion, your answer 1) seems to be the correct one.

Hope this helps!

**Comment by userzhi** [comment]
> Hi @userzhi, you point out a very adequate question. The paper discusses this briefly in the experiments/conclusion section. There seems to be a bottleneck caused by the RPN when the target objects look very different from the ones in the training dataset. You can clearly see this in the paper results. When training the RPN with DOTA, the results on SIMD are quite good, as the types of objects are subclasses of the DOTA categories (types of planes, vehicles, etc.). Contrary to this, DIOR contains several classes that have nothing to do with the ones in DOTA, e.g. toll station, wind turbine, dam, etc. These look very different and are considered as background by the FPN, so no quality bounding boxes are generated for them, causing a lot of missed detections.
> 
> In conclusion, your answer 1) seems to be the correct one.
> 
> Hope this helps!

@xavibou  thanks for your reply , this is very helpful for me
